package.json

```diff
@@ -83,6 +83,7 @@
     "start": "vite --port 5000",
     "storybook": "pnpm build && storybook dev -p 6006",
     "build-storybook": "pnpm build && storybook build --quiet -o preview",
+    "heroku-postbuild": "pnpm build-storybook && chmod +x scripts/setup-auth.sh && ./scripts/setup-auth.sh",
```

Comment:
[nitpick] We can commit the executable bit and drop `chmod +x scripts/setup-auth.sh` from, here